openedx/adg/lms/registration_extension/models.py

```diff
@@ -14,12 +14,13 @@ class ExtendedUserProfile(models.Model):
     """
 
     user = models.OneToOneField(
-        User, unique=True, db_index=True, related_name='extended_profile', on_delete=models.CASCADE)
+        User, db_index=True, related_name='extended_profile', on_delete=models.CASCADE)
 
-    birth_date = models.DateField(verbose_name=_('Birth Date'), null=True)
-    saudi_national = models.BooleanField(verbose_name=_('Saudi National'), null=True)
+    birth_date = models.DateField(verbose_name=_('Birth Date'), null=True, blank=True)
+    saudi_national = models.BooleanField(verbose_name=_('Saudi National'), null=True, blank=True)
```

Comment:
Shouldn't it be `blank=False`?

Reply:
No, Registration Form is also using this model. they are field are optional as they are not required in Registration Form.